src/bool.cr

```diff
@@ -55,4 +55,9 @@ struct Bool
   def to_s(io)
     io << to_s
   end
+
+  # Returns `self`
```

Comment:
I feel like the "Return `self`" comment everywhere should be a bit more verbose, pointing out that this is a `Value` type and thus assignment creates a copy.

Perhaps just "Returns `self`, assignment creates a copy, see `Value`."